src/asciidoc/docs/develop/workLocally.adoc

```diff
@@ -279,49 +251,22 @@ set KOJI_PROJECT_TOKEN=a6676f53-44fe-4109-819a-69df620ad7ed
 +
 TIP: Another alternative is to use the https://www.npmjs.com/package/dotenv[dotenv] package to read the values from the `.env` file.
 
-==== Starting the watcher on the backend (Windows only)
-
-. Install npm-run-all.
-+
-[source,bash]
-npm i --save-dev npm-run-all
-
-. Modify the `package.json` file.
-+
-[source,JSON]
-{
-  "scripts: {
-    ...
-    "start-dev": "NODE_ENV=development babel-watch -L --watch ../.koji/ src/server.js",
-    "start": "NODE_ENV=production node dist/server.js",
-    "watch": "koji-vcc watch",
-    "start-windows": "run-p watch start-dev"
-  }
-}
-
-. Remove any duplicate scripts.
-
-. Launch the backend with the following command.
-+
-[source,bash]
-npm run start-windows
-
 == Testing your local deployment
 
-. In the terminal windows, check that the frontend reported `Compiled successfully` and the backend reported `backend started`.
+. In the terminal windows, check that the frontend reported `Compiled successfully` and the backend reported `Server is listening on port 3333`.
 . If you see errors, make sure that no other applications are running on ports `8080` and `3333`.
 . In your browser, visit `\http://0.0.0.0:8080`.
 +
-You should see the Element Match game running.
-. Click *Leaderboard* to check that it is working.
+You should see the Falling Objects game running.
+. Click *Top Players* to check that the leaderboard is working.
 +
-If you haven't played the game yet, there will be no scores to show, but you will see the Top Scores title and a Close link that returns you to the Welcome screen.
-. Click *Start Matching*, and play the game.
+If you haven't played the game yet, there will be no scores to show, but you will see the Top Players title and a Close link that returns you to the Welcome screen.
+. Click the start game icon image:WL_01_start-game.jpg[Start game icon], and play the game.
```

Comment:
Let's remove this image. It's a bit big and I think the game is pretty easy to use.